packages/supabase/supabase/migrations/20260510201820_add_pds_handles_registry_and_community_bluesky.sql

```
@@ -0,0 +1,71 @@
+CREATE INDEX IF NOT EXISTS idx_pds_handles_entity
+  ON public.pds_handles (entity_type, entity_id);
+
+-- Index for DID lookups
+CREATE INDEX IF NOT EXISTS idx_pds_handles_did
+  ON public.pds_handles (did) WHERE did IS NOT NULL;
+
+-- Enable RLS
+ALTER TABLE public.pds_handles ENABLE ROW LEVEL SECURITY;
+
+-- Anyone can read handles (they're public identities)
+CREATE POLICY "Anyone can read handles"
```

SQLFluff (4.1.0)

[error] Do not use special characters in identifiers.

(RF05)